cinema/migrations/0001_initial_squashed_0002_movie_image.py

```diff
@@ -0,0 +1,94 @@
+# Generated by Django 4.1 on 2024-10-27 19:35
+
+import cinema.models
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    replaces = [('cinema', '0001_initial'), ('cinema', '0002_movie_image')]
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Actor',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('first_name', models.CharField(max_length=255)),
+                ('last_name', models.CharField(max_length=255)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='CinemaHall',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=255)),
+                ('rows', models.IntegerField()),
+                ('seats_in_row', models.IntegerField()),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Genre',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=255, unique=True)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Movie',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('title', models.CharField(max_length=255)),
+                ('description', models.TextField()),
+                ('duration', models.IntegerField()),
+                ('actors', models.ManyToManyField(to='cinema.actor')),
+                ('genres', models.ManyToManyField(to='cinema.genre')),
+                ('image', models.ImageField(null=True, upload_to=cinema.models.movie_image_upload_path)),

```

Comment:
Ensure that the `movie_image_upload_path` function is defined in `cinema.models` as it is used for the `upload_to` parameter in the `ImageField`.